features/old/accounts/bulk_operations.feature

```diff
@@ -17,28 +17,23 @@ Feature: Bulk operations
     Given current domain is the admin domain of provider "foo.3scale.localhost"
     Given I am logged in as provider "foo.3scale.localhost"
 
-  # FIXME: THREESCALE-7195 this scenario is failing in CircleCI. We need to refactor it as an integration test.
-  @wip
   Scenario: Show and hide bulk operations controls
     When I navigate to the accounts page
     When I check select for "bob"
-    Then "Bulk operations" should be visible
+    And checks if bulk-operations the div is visible
```

Comment:
I think cucumbers describe which should be the user experience, for that reason I think this step shouldn't include any reference to the `div` in it's name. I think cucumber already provide an `is visible` step. You can also use `I should see`/`I should not see`.

Reply:
Yes, it can check for a div internally, but the name shouldn't mention the div IMO, it should be something more closer to the user experience like `I should see the Bulk opertaions panel`

Reply:
I agree with what @jlledom says. And it's true that just rewording these two lines as `Then I should see "Bulk operations"` and `Then I should not see "Bulk operations"` should be enough, I think, without adding the new steps.

Also, the `When` online 22 can be changed to `And` for sake for nicer wording :)